R/tax_check.R

```diff
@@ -245,11 +248,11 @@ tax_check <- function(taxdf, name = "genus", group = NULL, dis = 0.05,
   err <- sp[!unlist(lapply(sp, is.null))]
   err <- as.data.frame(do.call(rbind, err))
   err$f1 <- as.vector(table(taxdf2[, "name"])[match(err$V1,
-                                                names(table(
-                                                  taxdf2[, "name"])))])
+                                                       names(table(
+                                                         taxdf2[, "name"])))])
   err$f2 <- as.vector(table(taxdf2[, "name"])[match(err$V2,
-                                                names(table(
-                                                  taxdf2[, "name"])))])
+                                                       names(table(
+                                                         taxdf2[, "name"])))])
```

Comment:
Should these have drop = TRUE?

Reply:
I think this is fine because taxdf2 is a `data.frame` generated in L181?